app/javascript/components/organization-teams-dropdown.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <teams-dropdown
+    :github-login="githubLogin"
+    :teams="this.organizationTeams">
+  </teams-dropdown>
+</template>
+
+<script>
+import TeamsDropdown from './teams-dropdown';
+import { mapState } from 'vuex';
+
+export default {
+  props: {
+    githubLogin: String,
+  },
+  components: {
+    TeamsDropdown,
+  },
+  computed: mapState({
+  	organizationTeams: state=> state.profile.organizationTeams,
```

Comment:
Mixed spaces and tabs                                 no-mixed-spaces-and-tabs<br>Missing space before =>                               arrow-spacing